interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -1113,8 +1110,7 @@ def test_fromCti(self):
 
     def test_fromXml(self):
         import xml.etree.ElementTree as ET
-        p = os.path.dirname(__file__)
-        root = ET.parse(pjoin(p, '..', 'data', 'h2o2.xml')).getroot()
+        root = ET.parse(str(self.cantera_data_path / "h2o2.xml")).getroot()
```

Comment:
Elsewhere, you didn't cast the argument to a string, so I don't think you have to here either.

Reply:
True. wasn't sure that `ElementTree` took `Path` as an argument.